includes/subscriptions/class-wc-payments-subscriptions-admin-notices.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Class WC_Payments_Subscriptions_Admin_Notices
+ *
+ * @package WooCommerce\Payments
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class for handling admin notices related to Stripe Billing deprecation.
+ */
+class WC_Payments_Subscriptions_Admin_Notices {
+
+	/**
+	 * Initialize the class and attach callbacks.
+	 */
+	public function __construct() {
+		add_action( 'admin_notices', [ $this, 'display_stripe_billing_deprecation_notice' ] );
```

Comment:
Probably, move this callback to `init_hooks` similar to other cases. Longer context here paJDYF-803-p2#comment-17834

https://github.com/Automattic/woocommerce-payments/blob/1ccba09010a77cb42fa6e38e4e75bc1d032a90fe/includes/class-wc-payments.php#L747-L751

Reply:
nice addressed here 1d98763c2